graphics/src/AssimpLoader_TEST.cc

```diff
@@ -697,3 +697,20 @@ TEST_F(AssimpLoader, LoadGlbPbrAsset)
   EXPECT_STREQ("Action2", skel->Animation(1)->Name().c_str());
   EXPECT_STREQ("Action3", skel->Animation(2)->Name().c_str());
 }
+
+/////////////////////////////////////////////////
+// Checks for null root node animation and valid
+// x displacement in non root node's animation.
+TEST_F(AssimpLoader, CheckNonRootDisplacement)
+{
+  common::AssimpLoader loader;
+  common::Mesh *mesh = loader.Load(common::testing::TestFile("data",
+        "walk.dae"));
+  auto meshSkel =  mesh->MeshSkeleton();
+  std::string rootNodeName = meshSkel->RootNode()->Name();
+  common::SkeletonAnimation *skelAnim = meshSkel->Animation(0);
+  common::NodeAnimation *rootNode = skelAnim->NodeAnimationByName(rootNodeName);
+  EXPECT_EQ(nullptr, rootNode);
+  auto xDisplacement = skelAnim->XDisplacement();
+  ASSERT_TRUE(xDisplacement);
```

Comment:
can we add another example below with that return xDisplacement=false?

Reply:
This might take a while, I need to understand how to remove animations from collada.

Reply:
@ahcorde,
Can I create just an empty animation and test the function with that?

Reply:
yes,

Reply:
Done, added new test file for skeletonAnimation because we are not using any mesh for the new test.

Reply:
@ahcorde ,
One question, I added new test file and re-compile everything using colcon, when I tried to make test, make didnt see any test, I had to remove gz-common build and re-compile everything. Took me 30min, is there a easier way? Refreshing cache for test?